src/routes.js

```diff
@@ -3,9 +3,11 @@ import BreedsList from "./components/BreedsList.vue";
 import NotFound from "./components/NotFound.vue";
 import BreedDetails from "./components/BreedDetails.vue";
 
+const isProduction = process.env.NODE_ENV === "production";
+
 const router = new VueRouter({
   mode: "history",
-  base: "/Vue-FindAdogBreed/",
+  base: isProduction ? "/Vue-FindAdogBreed/" : "/",

```

Comment:
Zdej sem spremenil `base`, da se spreminja glede na to ali je `production `ali `development`. Tudi ko deployam app načeloma dela, mi je le uspelo.  Gledal sem napačno konfiguracijo, zato mi v preteklosti ni hotelo prikazati strani, ko sem deplojal.  Moram samo še stestirati, če dejansko zamenja base za production oz development.